sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventBatch.cs

```diff
@@ -127,6 +128,7 @@ public override bool TryAdd(EventData eventData)
             Argument.AssertNotNull(eventData, nameof(eventData));
             Argument.AssertNotDisposed(_disposed, nameof(EventDataBatch));
 
+            eventData.PendingPublishSequenceNumber = MeasureUsingSequenceNumber;
```

Comment:
owner level and producer group id should also be here because they're a part of system properties.

Reply:
I'm going to punt that until I start on the service integration.